### ISSUE TITLE
AVT 1 - Content Switcher has DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4520]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to the React Component 
Run DAP and the following violations are present (see screenshot):
<img width="1210" alt="Screen Shot 2019-11-01 at 10 06 17 AM" src="https://user-images.githubusercontent.com/21676914/68034436-ab8a4800-fc8f-11e9-952e-2407c78b8cf8.png">
